homeassistant/components/unifi/device_tracker.py

```diff
@@ -214,6 +190,28 @@ def __init__(self, client, controller):
         self.client = client
         self.controller = controller
 
+    @property
+    def entity_registry_enabled_default(self):
+        """Return if the entity should be enabled when first added to the entity registry."""
+        if not self.controller.option_track_clients:
+            return False
+
+        if (
+            not self.client.is_wired
+            and self.controller.option_ssid_filter
+            and self.client.essid not in self.controller.option_ssid_filter
+        ):
+            return False
+
+        if not self.controller.option_track_wired_clients and self.client.is_wired:
+            return False
+
+        return True
+
+    async def async_added_to_hass(self):
```

Comment:
Shouldn't you register here for updates?

Reply:
It only updates the entity if it has an entity id right now.

I would need to redo how tracked works then. Perhaps if I put it in the controller instead?

Reply:
Update is only called after verifying that entity registry entry is not disabled